test/integration-test.py

```diff
@@ -53,6 +55,18 @@ def setup_seventy_days_ago():
     global old_authzs
     _, old_authzs = auth_and_issue([random_domain()])
 
+def setup_twenty_days_ago():
+    """Do any setup that needs to happen 20 day in the past, for tests that
+       will run in the 'present'.
+    """
+    # Issue a certificate with the clock set back, and save the authzs to check
+    # later that they are valid (200). They should however require rechecking for
```

Comment:
Let's move this comment into the docstring. Also, can you add a short explanation of why we save the client as well as the authzs?

Reply:
I can do this, but before I do, it is worth noting that this is the way that all of these functions (setup_X_days_ago()) are written - the docstring describes the high level behavior, then inline comments detail the more specific details. Please let me know if you want to change all three functions.